### PR TITLE
add ilok-license-manager

### PR DESCRIPTION
**Pull request by yutarody**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/ilok-license-manager.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url 'http://installers.ilok.com/iloklicensemanager/LicenseSupportInstallerMac.zip'
+  name 'iLok License Manager'
+  homepage 'https://ilok.com/#!license-manager'
+  license :gratis
+
+  container nested: "LicenseSupportInstallerMac_v#{version}.dmg"
+
+  pkg 'License Support.pkg'
+
+  uninstall script: 'Uninstall License Support.app/Contents/MacOS/Application Stub'
```
Comment (vitorgalvao) [inline Casks/ilok-license-manager.rb]: Won’t this call a GUI?

Comment (yutarody) [inline Casks/ilok-license-manager.rb]: Uninstall script is Automator Application.
So for entering a password GUI will Invoked during the uninstallation.

Comment (vitorgalvao) [inline Casks/ilok-license-manager.rb]: We can’t accept it, then.

Take a look at our [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md) and use something else (like `pkgutil`).

**Comment by yutarody** [comment]
Thank you.I have changed the uninstall method.

File: Casks/ilok-license-manager.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  url 'http://installers.ilok.com/iloklicensemanager/LicenseSupportInstallerMac.zip'
+  name 'iLok License Manager'
+  homepage 'https://ilok.com/#!license-manager'
+  license :gratis
+
+  container nested: "LicenseSupportInstallerMac_v#{version}.dmg"
+
+  pkg 'License Support.pkg'
+
+  uninstall pkgutil: 'com.paceap.pkg.eden.iLokLicenseManager'
```
Comment (vitorgalvao) [inline Casks/ilok-license-manager.rb]: See how other casks do it.

`pkgutil` should be an array, and `uninstall` should not be repeated.

Comment (yutarody) [inline Casks/ilok-license-manager.rb]: I'm sorry. Many times made a mistake. And fix later.

**Comment by yutarody** [comment]
I have changed the pkgutil to the array. Thank you.

**Pull request merged**